tests/components/nextdns/__init__.py

```diff
@@ -52,6 +52,8 @@
     google_safe_browsing=False,
     idn_homograph_attacks_protection=True,
     logs=True,
+    logs_location="ch",
```

Comment:
Should this be something settable for the user?

Reply:
`logs retention` and `logs location` will be implemented as select entities in the next PR.